FVCalc.java

```diff
@@ -2,5 +2,11 @@
 public class FVCalc {
 	public static void main(String[] args){
 		// Replace this comment with your code
+		double a = Double.parseDouble(args[0]);
+		double b = Double.parseDouble(args[1]);
+		double c = Double.parseDouble(args[2]);
+		double d =b/100;
+		double a1=a*(Math.pow(1+d,c));
```

Comment:
Missing spacing between the value and the name of the variable
Lack of spacing between parameters of a function call:
Lack of spacing between mathematical operations.